src/MPDevice.cpp

```diff
@@ -2989,33 +3101,12 @@ bool MPDevice::checkLoadedNodes(bool checkCredentials, bool checkData, bool repa
             return_bool = false;
         }
 
-        for (auto &i: dataNodes)
-        {
-            if (!i->getPointedToCheck())
-            {
-                qWarning() << "Orphan data parent found:" << i->getService() << "at address:" << i->getAddress().toHex();
-                if (repairAllowed)
-                {
-                    addOrphanParentToDB(i, true, repairAllowed);
-                }
-                nbOrphanDataParents++;
-            }
-        }
-        QListIterator<MPNode*> i(dataChildNodes);
-        while (i.hasNext())
+        checkLoadedDataNodes(nbOrphanDataParents, nbOrphanDataChildren, repairAllowed, Common::DATA_ADDR_IDX);
```

Comment:
the tag pointednode check is doing at f (checkCredentials && !tagPointedNodes(checkCredentials, checkData, repairAllowed)) ?

can you explain why you specified false in the following call?

tagPointedNodes(false, true, repairAllowed, Common::CRED_ADDR_IDX, Common::NOTE_ADDR_IDX);

Reply:
> (checkCredentials && !tagPointedNodes(checkCredentials, checkData, repairAllowed)) ?
tagPointedNodes in this call is tagging pointed nodes for data (files) nodes, because the last parameter is Common::DATA_ADDR_IDX by default.

> 
> can you explain why you specified false in the following call?
> tagPointedNodes(false, true, repairAllowed, Common::CRED_ADDR_IDX, Common::NOTE_ADDR_IDX);
In this case we are tagging pointed nodes for Notes, the first parameter is tagCredentials, which we should not tag in this case.